backend/projects/serializers.py

```diff
@@ -143,6 +145,20 @@ def _has_siblings(self, obj) -> bool:
             > 1
         )
 
+    def _user_metrics(self, obj):
+        metrics = list(
+            MetricsPreferences.objects.filter(project=obj).filter(
```

Comment:
You can put both arguments in the same `filter`, with 
```
MetricsPreferences.objects.filter(project=obj, user_id=self.context.get("user_id")
```

Reply:
I think that here you would not get the unique `MetricsPreference` object that corresponds to `user` and `project`, but rather the list of `MetricPreferences` right ?